bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableTableAdminClientWrapper.java

```diff
@@ -306,4 +324,134 @@ public Void apply(Empty input) {
   public ApiFuture<Operation> createTableFromSnapshotAsync(CreateTableFromSnapshotRequest request) {
     return ApiFutureUtil.adapt(delegate.createTableFromSnapshotAsync(request));
   }
+
+  @Override
+  public ApiFuture<Backup> createBackupAsync(CreateBackupRequest request) {
+    ListenableFuture<Operation> backupAsync =
+        delegate.createBackupAsync(
+            request.toProto(instanceName.getProjectId(), instanceName.getInstanceId()));
+    Function<Operation, Backup> function =
+        new Function<Operation, Backup>() {
+          @Override
+          public Backup apply(final Operation operation) {
+            try {
+              Future<Operation> operationFuture =
+                  batchThreadPool.submit(
+                      new Callable<Operation>() {
+                        @Override
+                        public Operation call() throws Exception {
+                          return delegate.waitForOperation(operation);
+                        }
+                      });
+
+              return Backup.fromProto(
+                  operationFuture
+                      .get()
+                      .getResponse()
+                      .unpack(com.google.bigtable.admin.v2.Backup.class));
+            } catch (IOException | InterruptedException | ExecutionException e) {
+              throw new IllegalStateException(e);
```

Comment:
InterruptedException should be handled separately so that the Thread.interrupted flag can be preserved. Similar how waitForOperation() is implemented